src/com/tkjngine/gametable/monsterwounds/PogMenuRenderListener.java

```diff
@@ -0,0 +1,117 @@
+package com.tkjngine.gametable.monsterwounds;
+
+import com.galactanet.gametable.Pog;
+import com.galactanet.gametable.events.IPogMenuRenderEventListener;
+import com.galactanet.gametable.events.PogMenuRenderEvent;
+import com.galactanet.gametable.GametableFrame;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Objects;
+import java.util.HashMap;
+
+public class PogMenuRenderListener implements IPogMenuRenderEventListener
+{
+    private HashMap<Pog, PogAdapter> pogRegistry; // TODO: Pull this out of this class and contain separately
```

Comment:
I'll create an issue for this tech debt. For now just want to get the working feature in master.